examples/example_win32_directx11/main.cpp

```diff
@@ -97,11 +97,15 @@ int main(int, char**)
         if (done)
             break;
 
+        ImGui::UpdateFontDemo();
```

Comment:
seems like u forgotten to remove this, because it getting exposed only in imgui_demo.cpp, even as static

Reply:
anyway after my tries even without this line, it seems like dx11 example not working

Reply:
Please test with DX9. As it is the only one that was changed to match latest changes.